### ISSUE TITLE
Marshal change enterat&exitat is not correclty updated on the frontend when changing Rounds

### ISSUE THREAD

**Issue by Klaus-Michael** [RotorHazard/RotorHazard#142]
Changes to the enterat&exitat values are not correclty updated/saved when commiting changes.

steps to reproduce:
have at last two saved rounds
edit enterat for round 1
recalculate
commit changes
switch to round 2
switch back to round 1

enter&exit values are now back to the original values for round 1, not the new ones.
Refreshing the page shows the new values for round 1. looks as if the db entries are OK but some caching is going on while staying on the page

**Comment by HazardCreative** [comment]
Yes, the data is loaded from the server on page load and saving doesn't overwrite this local storage. I did find this useful during construction as it allowed me to easily compare and contrast the realtime and recalc results—which I was doing a lot before we got the accuracy issue solved. There's little use for it now, though, so I agree this should be cleaned up. I'm actually going to mark it as a bug because the implications of this can definitely lead to confusion.
